build.py

```diff
@@ -0,0 +1,53 @@
+from bs4 import BeautifulSoup
+from translation_dictionary import translations
+import shutil, os
+
+SRC = "templates"
+DST_EN = "en"
+DST_SO = "so"
```

Comment:
when there are more of these on multiple files, might be worth it to put all the constant into a `.env` file